student-info-system/sis/models.py

```diff
@@ -56,6 +56,119 @@ def has_professor(self):
             pass
         return has
 
+    # DEMOGRAPHIC DATA
+    AGE = (
+        ('Under 18', 'Under 18'),
+        ('18-21', '18-21'),
+        ('22-25', '22-25'),
+        ('26-30', '26-30'),
+        ('31-40', '31-40'),
+        ('41-54', '41-54'),
+        ('55-64', '55-64'),
+        ('65 or over', '65 or over'),
+        ('Decline to State', 'Decline to State'),
+    )
+    demo_age = models.CharField(verbose_name="Age Group", max_length=20, blank=True, choices=AGE)
+    RACE = (
+        ('White/Caucasian', 'White/Caucasian'),
+        ('Native Hawaiian or Pacific Islander', 'Native Hawaiian or Pacific Islander'),
+        ('Hispanic', 'Hispanic'),
+        ('Black', 'Black'),
+        ('American Indian/Alaska Native', 'American Indian/Alaska Native'),
+        ('Decline to State', 'Decline to State'),
+    )
+    demo_race = models.CharField(max_length=40, blank=True, choices=RACE)
+    GENDER = (
+        ('Male', 'Male'),
+        ('Female', 'Female'),
+        ('Trans', 'Trans'),
```

Comment:
Can we make this non-binary? it would encompass more people

Reply:
added